tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Packages/packages.spec.ts

```diff
@@ -46,9 +45,58 @@ test.describe('Packages', () => {
       .build();
     const generatedContent = await umbracoApi.content.save(rootContentNode);
     await CreatePackage(umbracoApi, generatedContent.Id);
-
   }
 
+  test('Creates a simple package', async ({page, umbracoApi, umbracoUi}) => {
+    await umbracoApi.packages.ensureNameNotExists(packageName);
+    await umbracoApi.content.deleteAllContent();
+    await umbracoApi.documentTypes.ensureNameNotExists(rootDocTypeName);
+
+    const rootDocType = new DocumentTypeBuilder()
+      .withName(rootDocTypeName)
+      .withAllowAsRoot(true)
+      .build();
+
+    const generatedRootDocType = await umbracoApi.documentTypes.save(rootDocType);
+    const rootDocTypeAlias = generatedRootDocType["alias"];
+
+    const rootContentNode = new ContentBuilder()
+      .withContentTypeAlias(rootDocTypeAlias)
+      .withAction("saveNew")
+      .addVariant()
+        .withName(nodeName)
+        .withSave(true)
+      .done()
+      .build();
+    await umbracoApi.content.save(rootContentNode);
+
+    // We have to wait for navigation to the packages section, if not it can cause the test to fail
+    await Promise.all([
+      page.waitForNavigation(),
+      umbracoUi.goToSection(ConstantHelper.sections.packages)
+    ]);
+    await page.locator('[data-element="sub-view-umbCreatedPackages"]').click();
+    await page.locator("button", {hasText: "Create package"}).click();
+
+    // Fill out package creation form
+    await page.waitForTimeout(2000);
```

Comment:
We should find better waits than just waiting for timeouts, can we maybe find some elements on the page, that signifies the page has loaded ? 🤔